mmdeploy/codebase/mmdet/models/dense_heads/rpn_head.py

```diff
@@ -145,7 +144,7 @@ def rpn_head__get_bboxes(ctx,
 
 
 @FUNCTION_REWRITER.register_rewriter(
-    'mmdet.models.dense_heads.RPNHead.get_bboxes', backend='ncnn')
+    'mmdet.models.dense_heads.RPNHead.get_bboxes', backend=Backend.NCNN.value)
```

Comment:
Do we need to instead all func backend=str to backend=Backend.XXX.value?

Reply:
We better do so, in case we might change the string in the future.
But not in this PR.